gnome-initial-setup/pages/live-chooser/gis-live-chooser-page.c

```diff
@@ -51,6 +51,19 @@ typedef struct _GisLiveChooserPagePrivate GisLiveChooserPagePrivate;
 G_DEFINE_TYPE_WITH_PRIVATE (GisLiveChooserPage, gis_live_chooser_page, GIS_TYPE_PAGE);
 
 static void
+disable_chrome_auto_download (GisLiveChooserPage *page)
+{
+  GError *error = NULL;
+
+  if (!gis_pkexec (DATADIR "/eos-google-chrome-helper/eos-google-chrome-system-helper.py",
+                   NULL, &error))
+    {
+      g_warning ("Failed to disable Chrome auto-download: %s\n", error->message);
```

Comment:
Hey @alepuccetti, since this is trivial, if you prepare a patch for upstream, Cosimo or I will commit it quickly. That way we don't wind up carrying an unimportant change downstream forever. I don't think it's important to remove these newlines downstream, except in our Endless patches.